index.js

```diff
@@ -1 +1,38 @@
-// implement your express server here
\ No newline at end of file
+var express = require('express');
+var trucks = require('./trucks');
+var truckRouter = require('./routes/truckRoutes');
+
+var app = express();
+
+app.use('/trucks', truckRouter);
+app.use(express.static('public'));
+
+
+app.get('/trucks', function(request, response){
```

Comment:
All of these routes should be removed. Once you use the truckRouter, it's handling all the requests to the `/trucks` endpoint.